RealTimeFaceMaskDetector/TCPServer/WindowsService.cpp

```diff
@@ -120,7 +120,15 @@ bool Service::Initialize(const unsigned short argc, const wchar_t* const argv[])
 
 void Service::StartServerWork(bool& is_listening_started)
 {
-	std::thread listen_multiple_clients([&]() {s_socket_server.StartListening(is_listening_started); });
+	std::thread listen_multiple_clients([&]() 
+		{
+			s_instance->s_instance->s_socket_server->StartListening(is_listening_started);
```

Comment:
Why do we need to get s_instance from s_instance?? How it should works?

Reply:
A typo. I corrected it